README.md

```diff
@@ -22,3 +22,7 @@ Scratchy, application de tchat projet LPH.
   - [📸 Frontend diagram](doc/front-uml-class-diagram.png)
 
 
+## Tools
+
+ - :wrench: tools documentation directory is [here](doc/tools)
+ - ![python emoji](logos/python.png "python") python linter: [flake8](doc/tools/flake8.md)
```

Comment:
It might be better to move this documentation inside the server folder as it concern only the server ?

Reply:
there will be linter for front too